noir/noir-repo/acvm-repo/bn254_blackbox_solver/src/fixed_base_scalar_mul.rs

```diff
@@ -118,7 +134,7 @@ mod grumpkin_fixed_base_scalar_mul {
         let invalid_limb = max_limb + FieldElement::one();
 
         let expected_error = Err(BlackBoxResolutionError::Failed(
-            BlackBoxFunc::FixedBaseScalarMul,
+            BlackBoxFunc::VariableBaseScalarMul,
```

Comment:
```suggestion
            BlackBoxFunc::FixedBaseScalarMul,
```
After mapping the error these would then need to be switched back

Reply:
You can then add a separate call to `variable_base_scalar_mul` in these failure tests where the error is `VariableBaseScalarMul`